app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+
+  def new
+  end
+
+  def create
+      user = User.find_by(email: params[:email])
+      if user && user.authenticate(params[:password])
+        session[:user_id] = user.id
+        flash[:notice] = "You are now logged in as #{user.first_name}"
+        redirect_to "/profile"
+      else
+        flash[:notice] = "The credentitals you have entered are invalid"
+        redirect_to "/login"
+    end
+  end
+
+  def index
```

Comment:
This doesn't belong here, I will find a way to take this out during a refactor

Reply:
I'm not totally sure what you are referring to, but looking back at Monster Shop we used a destroy method in our sessions controller for logout. We also had a line after `session.delete` that set `current_user = nil` and I'm sure there was a reason for that! I think if you just change index to destroy then no refactor needed.